rawpy/_rawpy.pyx

```diff
@@ -704,11 +705,27 @@ cdef class RawPy:
     property white_level:
         """
         Level at which the raw pixel value is considered to be saturated.
+        This value is calculated from the data for most cameras, and hardcoded for others.
         """
         def __get__(self):
             self.ensure_unpack()
             return self.p.imgdata.rawdata.color.maximum
 
+    property camera_white_level_per_channel:
+        """
+        Per-channel linear data maximum read from file metadata.
```

Comment:
I'd also simplify this (no need to copy/paste from LibRaw), and make terminology consistent within rawpy. Like: "Per-channel saturation level read from raw file metadata, if it exists. None otherwise."

Reply:
Sounds good. Will update this tomorrow.